pkg/eventing/v1alpha1/client.go

```diff
@@ -170,7 +170,7 @@ func (b *TriggerBuilder) Broker(broker string) *TriggerBuilder {
 	return b
 }
 
-func (b *TriggerBuilder) AddFilter(key, value string) *TriggerBuilder {
+func (b *TriggerBuilder) AddSingleFilter(key, value string) *TriggerBuilder {
```

Comment:
I'd suggest to keep `AddFilter` and jut add a `Filters` method for setting the full field (like for the other methods)

Reply:
In the latest version, I keep only Filters to add multiple filter.